lib/Doctrine/ORM/Proxy/Factory/StaticProxyFactory.php

```diff
@@ -127,6 +138,15 @@ public function getProxy(string $className, array $identifier) : GhostObjectInte
 
         $persister->setIdentifier($proxyInstance, $identifier);
 
+        foreach ($toManyFields as $toMany) {
+            $collection = $toMany->wrap($proxyInstance, null, $this->entityManager);
+
+            $collection->setInitialized(false); // @TODO not the right approach, IMO
```

Comment:
Seems correct, although I'd prefer the `$toMany` `Property` to do this for me :-\

Reply:
shouldn't this be done in `->wrap()` instead ? It looks weird to me that wrapping `null` gives you an initialized PersistentCollection.

Reply:
Yeah, TODO in fact. `wrap` was not working as expected either.